docs/api/oauth.md

```diff
@@ -0,0 +1,447 @@
+
+Once you have the token, you must send it to the following Audius API endpoint in order to verify that a) the signature was signed by the Audius user who completed the authentication, and b) the content of the token hasn't been tampered with. Upon verifying the validity of the JWT, the endpoint will return the authenticated user's decoded profile information.
+
+<ins>GET `/v1/users/verify_token?token=[JWT]`</ins>
+
+**Params**
+
+- token `string` - the JWT from the authentication flow that you would like to verify
+
+**Sending the request**
+
+To use the API, you first select an API endpoint from the list of endpoints returned by:
+
+`GET https://api.audius.co`
+
+Once you've selected a host, all API requests can be sent directly to it. For instance, if you picked this host:
+`https://audius-dp.singapore.creatorseed.com`, you would issue the verify token request to `https://audius-dp.singapore.creatorseed.com/v1/users/verify_token?token=<JWT>`, where `<JWT>` is replaced with the JWT you retrieved in the auth flow.
+
+We recommend selecting a host each time your application starts up as availability may change over time.
+
+**Success response**
+
+- Code: 200 OK
+- Content: The decoded JWT payload, which contains the user's profile information:
+
+```typescript
+{
+  userId: number, // unique Audius user identifier
+  email: string,
+  name: string, // user's display name
+  handle: string,
+  verified: boolean, // whether the user has the Audius "verified" checkmark
+
+  /** URLs for the user's profile picture, if any.
+  * If the user has a profile picture, three sizes will be available: 150x150, 480x480, and 1000x1000.
+  * If the user has no profile picture, this field will be empty.
+  */
+  profilePicture: {"150x150": string, "480x480": string, "1000x1000": string } | { misc: string } | undefined | null
+  sub: number, // alias for userId
+  iat: string, // timestamp for when auth was performed
+}
+```
+
+**Error responses**
+
+<ins>Invalid signature</ins>:
+
+- Code: `404` Not Found
+- Content: Error message describing the issue that occurred, e.g. "The JWT signature is invalid - wallet could not be recovered."
+
+<ins>Problem with `token` input</ins>:
+
+- Code: `400` Bad Request
+- Content: Error message, e.g. "the JWT signature could not be decoded."
+
+<br />
+
+### 3. Done!
+
+Once you've verified the JWT, the authentication flow is complete and you now have your user's Audius profile information.
+
+### Addendum: A quick note on email
+
+Once you know your user's Audius user id, you can retrieve their Audius information at any time using our SDK or web APIs. However, the one piece of profile information that is not available outside of the Log in with Audius response is the user's email address. If you do not store the user's email address after you receive it in the Log in with Audius success response, you can only re-retrieve the email through having the user re-complete the Log in with Audius flow.
```

Comment:
Could we maybe link to the existing section rather than duplicating it here?

Reply:
ohh ya that makes sense